examples/siro_sandbox/test_sandbox_app.py

```diff
@@ -0,0 +1,79 @@
+import os
+from unittest.mock import patch
+
+import magnum as mn
+import pytest
+from sandbox_app import (
+    DEFAULT_CFG_PATH,
+    SandboxDriver,
+    get_sandbox_app_args_parser,
+    override_config,
+)
+from utils.gui.gui_input import GuiInput
+from utils.gui.replay_gui_app_renderer import ReplayGuiAppRenderer
+
+from habitat_baselines.config.default import get_config as get_baselines_config
+
+
+@pytest.mark.skipif(
+    not os.path.exists("data/hab3_bench_assets"),
+    reason="This test requires hab3_bench_assets assets.",
+)
+@patch("utils.gui.replay_gui_app_renderer.ReplayRenderer", spec=True)
+@patch("utils.gui.replay_gui_app_renderer.ImageFramebufferDrawer", spec=True)
+@patch("utils.gui.replay_gui_app_renderer.TextDrawer", spec=True)
```

Comment:
Patching these thing until https://app.asana.com/0/home/1205684818278375/1205279290945037 is done.